modules/library/referencing/src/main/java/org/geotools/measure/SexagesimalConverter.java

```diff
@@ -77,13 +75,28 @@ private SexagesimalConverter(final int divider, final SexagesimalConverter inver
         this.inverse = inverse;
     }
 
+    @Override
+    public int compareTo(UnitConverter o) {
+        // TODO: ?
+        return 0;
+    }
+
     /** Returns the inverse of this converter. */
-    public final SexagesimalConverter inverse() {
+    public final SexagesimalConverter inverseWhenNotIdentity() {
         return inverse;
     }
 
+    @Override
+    protected boolean canReduceWith(AbstractConverter that) {
+        return that instanceof Inverse;
+    }
+
+    public Number convertWhenNotIdentity(Number value) {
+        return degreeToSexagesimalDegrees(value.doubleValue());
+    }
+
     /** Performs a conversion from fractional degrees to sexagesimal degrees. */
-    public double convert(double value) {
+    public double degreeToSexagesimalDegrees(double value) {
```

Comment:
public API break, should be documented on the upgrade guide at a minimum. For best results, deprecate the existin implementation and make it delegate to the new method.

Reply:
Actually this class extends AbstractConverter which implements the convert method and uses a final modifier.  This method now calls convertWhenNotIdentity() of this class.  This being said I see no need to document this on the upgrade guide.

Reply:
The logic is that convert(value) is still available, supplied by the superclass; so no net api change.